BaseClasses.py

```diff
@@ -593,8 +593,7 @@ def location_condition(location: Location):
         def location_relevant(location: Location):
             """Determine if this location is relevant to sweep."""
             if location.progress_type != LocationProgressType.EXCLUDED \
-                and (location.player in players["locations"] or location.event
-                     or (location.item and location.item.advancement)):
+               and (location.player in players["locations"] or location.advancement):
```

Comment:
I think this is how I would format this:

```python
            if location.progress_type != LocationProgressType.EXCLUDED and (
                location.player in players["locations"] or location.advancement
            ):
```

Not saying this is better, just presenting another option in case you were looking for one.

Reply:
I think I'd rather leave it how it is due to it being the current style in AP and I don't think we ever decided on which one we should use going forward.